incubator/bitcoind/templates/svc.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "bitcoind.fullname" . }}
+  labels:
+    app: {{ template "bitcoind.name" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  ports:
+    - name: rpc
+      port: {{ .Values.rpcPort }}
+      targetPort: rpc
+  selector:
```

Comment:
What about adding a tag in values for a selector here that you could add more nodes behind? Like

```
selector:
    bitcoind-rotation: {{ .Values.service.selector }}
```

This could be set on other instances of this chart and would allow for easy load balanced instances without a stateful set.